modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/jaxrs/exception/mapper/BaseExceptionMapper.java

```diff
@@ -81,8 +81,10 @@ private Problem _getSanitizedProblem(T exception) {
 			_log.warn("Problem " + problem, exception);
 		}
 
-		problem.setDetail(null);
-		problem.setTitle(null);
+		if (problem.getStatus() != Response.Status.NOT_ACCEPTABLE) {
```

Comment:
I think avoiding this code when sanitize is true makes no sense as we are sanitizing the problem. Maybe better printing the log trace in your child exception mapper